kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/JsonNodeUtil.java

```diff
@@ -0,0 +1,96 @@
+package com.provectus.kafka.ui.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.provectus.kafka.ui.exception.UnprocessableEntityException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+public class JsonNodeUtil {
+  private static final String NOT_OBJECT_EXCEPTION_MESSAGE = "JsonNode isn't Object";
+  private static final String NOT_ARRAY_EXCEPTION_MESSAGE = "JsonNode isn't Array";
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  public static Map<String, String> toMap(JsonNode node) {
+    if (node.isObject()) {
+      List<String> keys = getJsonObjectKeys(node);
+      List<String> values = getJsonObjectValues(node);
+      return IntStream.range(0, keys.size()).boxed()
+              .collect(Collectors.toMap(keys::get, values::get));
+    }
+    throw new UnprocessableEntityException(NOT_OBJECT_EXCEPTION_MESSAGE);
+  }
+
+  public static List<String> toList(JsonNode node) {
+    if (node.isArray()) {
+      return getStreamForJsonArray(node).map(JsonNode::toString).collect(Collectors.toList());
+    }
+    throw new UnprocessableEntityException(NOT_OBJECT_EXCEPTION_MESSAGE);
+  }
+
+
+
+  public static List<String> getJsonObjectKeys(JsonNode node) {
+    if (node.isObject()) {
+      return StreamSupport.stream(
+          Spliterators.spliteratorUnknownSize(node.fieldNames(), Spliterator.ORDERED), false
+      ).collect(Collectors.toList());
+    }
+    throw new UnprocessableEntityException(NOT_OBJECT_EXCEPTION_MESSAGE);
+  }
+
+  public static List<String> getJsonObjectValues(JsonNode node) {
+    if (node.isObject()) {
+      return getJsonObjectKeys(node).stream().map(key -> node.get(key).asText())
```

Comment:
in case of array or object values this will return empty strings, not sure that it is good behaviour

Reply:
sure, fixed it.